solutions/devsprint-michael-douglas-2/DeliveryAppTests/SnapshotTests/DividerViewSnapshotTests.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  DividerViewSnapshotTests.swift
+//  DeliveryAppTests
+//
+//  Created by Murillo R. Araújo on 22/02/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import DeliveryApp
+
+class DividerViewSnapshotTests: XCTestCase {
+
+    let snapshotSize = CGSize(width: 200, height: 100)
+
+    func testDividerView() {
```

Comment:
Uma coisa que esqueci de comentar no outro PR, mas que irei passar com vocês amanhã, é em relação a boas práticas de nomenclatura de Testes Unitários. Seguindo as boas práticas, o correto seria o seu método se chamar:

```suggestion
    func test_dividerView_whenSnapshot_shouldSucceed() {
```

Reply:
Boa, vou adotar essa prática